plutus-core/plutus-core.cabal

```diff
@@ -693,6 +693,18 @@ benchmark cost-model-test
     other-modules:
         CreateBuiltinCostModel
 
+executable print-cost-model           
+    import: lang
+    main-is: Main.hs
+    hs-source-dirs: cost-model/print-cost-model
+    ghc-options: -threaded -rtsopts -with-rtsopts=-N -Wall -Wincomplete-uni-patterns
+                 -Wincomplete-record-updates -Wredundant-constraints -Widentities
```

Comment:
```suggestion
```

Reply:
Yes, I suppose that for something like this we don't need all of this stuff: I just copied it in mindlessly.

Reply:
I think it's good to be consistent. At the moment the policy is to put this stuff everywhere, it's much harder if we are more inconsistent.

But the warnings should be in a common stanza!

Reply:
OK, I'll put it back! Someone should make an issue to put the warnings in a common stanza...

Reply:
They should already be in the common stanza, yeah.